modules/account-lib/src/coin/baseCoin/enum.ts

```diff
@@ -29,6 +29,10 @@ export enum TransactionType {
   StakingWithdraw,
   // Handle smart contract calls
   ContractCall,
+  // Wallet owner opt-in to receive this token in his wallet
```

Comment:
I think here will be good to provide a more generic documentation not only the valid case for algo.

Reply:
hope, it is better now